rpcs3/rpcs3qt/user_manager_dialog.cpp

```diff
@@ -99,6 +101,7 @@ void user_manager_dialog::Init(const std::string& dir)
 	connect(push_close, &QAbstractButton::clicked, this, &user_manager_dialog::close);
 	connect(push_remove_user, &QAbstractButton::clicked, this, &user_manager_dialog::OnUserRemove);
 	connect(push_create_user, &QAbstractButton::clicked, this, &user_manager_dialog::OnUserCreate);
+	connect(push_rename_user, &QAbstractButton::clicked, this, &user_manager_dialog::OnUserRename);
```

Comment:
also add a connect that enables and disables the pushbuttons, depending on selected user